### PR TITLE
ddl script typo fix

### PR DESCRIPTION
**Pull request by d-h-k**
## Description
- In this PR, two typos in the fosslight_create.sql file used for initial setting of the development environment have been corrected.

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## 🧑🏻‍💻ps
- Actually, this PR is my first contribution to this open source project. If there is something I don't know or am doing wrong, please feel free to give me feedback.

### CODE REVIEW

File: db/initdb.d/fosslight_create.sql [hunk 1]
```diff
@@ -898,7 +898,7 @@ CREATE TABLE IF NOT EXISTS `NVD_DATA_CONFIGURATIONS` (
 	INDEX `PRODUCT` (`PRODUCT`) USING BTREE,
 	INDEX `VERSION` (`VERSION`) USING BTREE,
 	INDEX `VENDOR` (`VENDOR`) USING BTREE,
-	INDEX `CRITERIA` (`CRITERIA`) USING BTREE,
+	INDEX `CRITERIA` (`CRITERIA`) USING BTREE
```
Comment (d-h-k) [inline db/initdb.d/fosslight_create.sql:901]: I think an error occurs if there is a comma at the end of the column.

File: db/initdb.d/fosslight_create.sql [hunk 2]
```diff
@@ -942,7 +942,7 @@ CREATE TABLE IF NOT EXISTS `OSS_DISCOVERED_SND_EMAIL` (
 	`CVE_ID` VARCHAR(16) NULL DEFAULT NULL,
 	`CVSS_SCORE` FLOAT(3,1) NULL DEFAULT NULL,
 	`SND_YN` VARCHAR(1) NULL DEFAULT 'N',
-	`SND_DATE` DATETIME NULL DEFAULT,
+	`SND_DATE` DATETIME NULL DEFAULT now(),
```
Comment (d-h-k) [inline db/initdb.d/fosslight_create.sql:945]: In my opinion~~~ it needs to be set a default value

**Comment by Min-Kyungsun** [comment]
Thank you for your contribution :)

**Comment by d-h-k** [comment]
@Min-Kyungsun  thank you merge my code! 

and.. one more question i have! 

Minor fixes VS Modify things registered as issues 

I wonder which of these two would be appropriate for an open source beginner to try. The issue is looking at this

**Comment by Min-Kyungsun** [comment]
> @Min-Kyungsun thank you merge my code!
> 
> and.. one more question i have!
> 
> Minor fixes VS Modify things registered as issues
> 
> I wonder which of these two would be appropriate for an open source beginner to try. The issue is looking at this #808

I recommend to try "good first issue" first. :)

**Pull request merged**